packages/backend/server/src/core/auth/controller.ts

```diff
@@ -52,6 +52,8 @@ export class AuthController {
   ) {}
 
   @Public()
+  @UseBasicGuard()
+  @Guard('captcha')
```

Comment:
looks verbose and we can't register more than one guard implementation for basic guard. I think it could be merged as `@CaptchaGuard`

Reply:
now merged into single decorator